r/Dockerfile

```diff
@@ -24,8 +24,10 @@ RUN R -e 'BiocManager::install(c("AnnotationDbi", "BiocGenerics", "GO.db", "pcaM
 
 RUN R -e 'install.packages("RcppSpdlog")'
 RUN R -e 'devtools::install_github("kharchenkolab/pagoda2@v0.1.4")'
-RUN R -e 'devtools::install_github("kharchenkolab/conos@v1.3.1")'
+#RUN R -e 'devtools::install_github("kharchenkolab/conos@v1.3.1")'
```

Comment:
if this is no longer going to be used, just delete it.

Reply:
I have just pushed it again